### PR TITLE
#33. Add target params to build step

### PR DESCRIPTION
**Pull request by dmitrii-snytkin**

### CODE REVIEW

**Review by AnBucyk — COMMENTED** [review]
Looks good to me, but I'd like Jimmy to take a look as well

File: src/taco-remote/lib/buildManager.ts [hunk 1]
```diff
@@ -150,6 +151,7 @@ class BuildManager {
             params.buildNumber = buildNumber;
             params.options = options;
             params.logLevel = logLevel;
+            params.target = req.query.target;
```
Comment (MSLaguana) [inline src/taco-remote/lib/buildManager.ts]: Isn't this a no-op since `params = req.query`, so `params.target` is identical to `req.query.target` already?

**Comment by MSLaguana** [comment]
The change here looks fine, but I'm not sure what it's trying to achieve. Is the goal to also pass in the equivalent of `--target=iPhone-4s` or similar to the cordova build command? Does that have an impact on the build step as opposed to the run/emulate step?

**Comment by dmitrii-snytkin** [comment]
If we don't specify target in build step, then this application cannot be run only on iPhone-5 emulator. Other emulators (at least, which I have tested) work fine without target params. So I have decided to set target for all builds

**Comment by MSLaguana** [comment]
Huh, interesting. In that case, this looks fine to me.

**Review by MSLaguana — APPROVED** [review]

File: src/taco-remote/lib/buildManager.ts (hunk 1 above)
Comment (dmitrii-snytkin) [inline src/taco-remote/lib/buildManager.ts]: Yes, I agree, I will delete it

**Pull request merged**